core/transaction_pool/impl/pool_moderator_impl.cpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "transaction_pool/impl/pool_moderator_impl.hpp"
+
+using kagome::common::Buffer;
+
+namespace kagome::transaction_pool {
+
+  PoolModeratorImpl::PoolModeratorImpl(std::shared_ptr<SystemClock> clock,
+                                       SystemClock::Duration ban_for,
+                                       size_t expected_size)
+      : ban_for_{ban_for},
+        clock_{std::move(clock)},
```

Comment:
Please, make order of the arguments the same as they are in the ctor

Reply:
ban_for cannot go before clock because ban_for has a default value and clock doesn't. But I think that now that I have an implementation of Clock suitable for PoolModerator, I can just give it a default argument as well.